.storybook/recipes/MenuWithIconButton.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+
+import { Icon } from '../../src';
+import { Menu, type MenuProps } from '../../src/components/Menu/Menu';
+
+export default {
+  title: 'Recipes/MenuWithIconButton',
+  component: Menu,
+  parameters: {
+    badges: [BADGE.BETA],
+    layout: 'centered',
+  },
+} as Meta<MenuProps>;
+
+export const WithVerticalDotsButton: StoryObj<MenuProps> = {
+  args: {
+    children: (
+      <>
+        <Menu.Button
+          className="!border-transparent-white-0 !bg-transparent-white-0 hover:!bg-button-secondary-neutral-background-hover"
+          showExpandIcon={false}
+          variant="icon"
+        >
+          <Icon name="dots-vertical" purpose="informative" title="show more" />
+        </Menu.Button>
```

Comment:
Related to that other comment, how might we create a sub-component that allows for this bit of code to be much briefer, and specify the proper prop values such that we avoid the boilerplate of lines 22-24

Reply:
we should especially not need people to pass extra styling to get this case working 😬

Reply:
Only needs the hover styling now with PlainButton